rocAL/include/pipeline/master_graph.h

```diff
@@ -55,6 +55,7 @@ THE SOFTWARE.
 #define MAX_SSD_ANCHORS 8732          // Num of bbox achors used in SSD training
 #define MAX_MASK_BUFFER 10000
 #define MAX_RETINANET_ANCHORS 120087  // Num of bbox achors used in Retinanet training
+#define MAX_ASCII_BUFFER 10000
```

Comment:
how did we come up with 10000?

Reply:
This is an arbitrary value for max number of ASCII characters in a particular file (like a .txt or .json).